src/persist-client/src/lib.rs

```diff
@@ -121,6 +133,9 @@ pub const BUILD_INFO: BuildInfo = build_info!();
 // Re-export for convenience.
 pub use mz_persist_types::{PersistLocation, ShardId};
 
+pub use crate::internal::encoding::Schemas;
+pub use crate::internal::state::HollowBatch;
```

Comment:
You might object to these being exposed. We only need `HollowBatch` because we use it to sniff out the size/num_rows from stashed batches without turning them into a `Batch` and/or reading them.

Reply:
`Batch::encoded_size_bytes` seems fine, yeah!